### PR TITLE
Add remaining course overview

### LINKED ISSUE #124: Finish adding course overviews
#### Describe Problem
A bunch of course overviews are there but not all of them
#### Suggest Changes
add em all
#### Provide Examples
https://uwindsorcss.github.io/wiki/courses/COMP-1000/overview

### LINKED ISSUE #134: Add course overviews for math courses
#### Describe Problem
Most CS programs require a bunch of math courses and we don't have overviews for them

#### Suggest Solution
Add course overviews similar to those we have for CS courses

#### Additional Details
Math courses should probably go at the bottom
See screenshot for reference
image

### PR DESCRIPTION
**Pull request by Lillliant**
### What are you trying to accomplish?

Add the remaining course overview articles.

### How are you accomplishing it?

I wrote a python program that generated these articles based on the information in course.json and Jeremie's UWin API.

### Is there anything reviewers should know?

I think the sidebar is too cluttered with this many courses, so I put them into folders depending on whether they are CS courses and what year-level they're in.

### Checks before you create your pull request

-   [x] Did you read and follow article requirements?
-   [ ] Did you run prettier? I tried to format them like the previous articles so it probably will be fine
-   [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

**Comment by jere-mie** [comment]
Open data to the rescue 😎

File: courses/COMP-2067/Overview.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+slug: /COMP-2067/overview
+---
+
+## Course Title
+
+The title for COMP-2067 is "Programming for Beginners".
+
+## Course Description
+
+This course introduces fundamental computer programming principles and structured programming concepts, with an emphasis on good programming. Stages of the software development cycles are introduced: analysis, design, implementation, debugging and deployment. May not be used to fulfill the major requirements of any major or joint major in Computer Science.) (3 lecture hours).
```
Comment (prairir) [inline courses/COMP-2067/Overview.md]:
```suggestion
This course introduces fundamental computer programming principles and structured programming concepts, with an emphasis on good programming. Stages of the software development cycles are introduced: analysis, design, implementation, debugging and deployment. (May not be used to fulfill the major requirements of any major or joint major in Computer Science.) (3 lecture hours).

```

**Review by prairir — CHANGES_REQUESTED** [review]
LGTM 🚀 🚀 . Please do that single change.

**Review by prairir — APPROVED** [review]
LGTM 🚀 🚀

**Pull request merged**